pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/BaseBrokerRequestHandler.java

```diff
@@ -197,8 +202,9 @@ public BrokerResponse handleRequest(JsonNode request, @Nullable RequesterIdentit
     }
 
     // Validate the request
+    FilterQueryTree[] filterQueryTree = new FilterQueryTree[1];
```

Comment:
Why does this need to be an array of 1?

Reply:
We need an instance of FilterQueryTree to be returned if it's been calculated. And simply returning this instance out of the method `validateRequest` looks unreasonable. Thus, I use an array to return the instance. Another way to handle that is to add a private variable of `FilterQueryTree` in requestParams.

Reply:
Updated the PR by returning request info.